### PR TITLE
Hide empty option groups

### PR DESCRIPTION
**Pull request by bmcorum**
- Add new setting for hiding of empty option groups (default false)
- Set title on search input for 508 compliance
- Update package version
- Rebuild javascript files

### CODE REVIEW

**Comment by bmcorum** [comment]
@fabianlindfors any issues with this latest pull request?

File: src/multi.js [hunk 1]
```diff
@@ -168,6 +168,16 @@ var multi = (function() {
         }
       }
     }
+
+    // Hide empty optgroups
+    if (settings.hide_empty_groups) {
+      var optgroups = document.getElementsByClassName('item-group');
+      for (var i = 0; i < optgroups.length; i++) {
+        if (optgroups[i].childElementCount < 2) { // only has a
```
Comment (fabianlindfors) [inline src/multi.js]: Could you complete this comment? And perhaps put it above the line instead?

**Review by fabianlindfors — COMMENTED** [review]
Sorry, Brian! This PR slipped my mind. Everything looks good, just a small fix and then we're ready to merge :)

**Comment by bmcorum** [comment]
@fabianlindfors sorry about the comment. I either never finished my thought or deleted it by accident. I decided after reviewing your code base that it would be best to have this setting set to false so that anyone who is using a previous version could upgrade without having to change any settings. I have a few more changes I have been contemplating, mostly around accessibility and compliance. Any preferences as far as coding standards? I will try to follow what I see in the current code base.

**Comment by fabianlindfors** [comment]
No worries, Brian! Really thankful for the work you're putting in. I don't have much time these days to spend on multi.js but I'm happy that it keeps being useful.

Improvement to accessibility and compliance would be very welcome! When I created the project I didn't follow any coding standard myself so just try to stay consistent with the rest, not that it's particularly good. I should probably introduce some standardized formatter and linter some day!

**Pull request merged**